bindings/python/iota_sdk/types/unlock.py

```diff
@@ -74,3 +75,33 @@ class NftUnlock:
     """
     reference: int
     type: int = field(default_factory=lambda: int(UnlockType.Nft), init=False)
+
+
+def unlock_from_dict(d: Dict[str, Any]) -> Union[SignatureUnlock, ReferenceUnlock, AccountUnlock, NftUnlock]:
+    """
+    Takes a dictionary as input and returns an instance of a specific class based on the value of the 'type' key in the dictionary.
+
+    Arguments:
+    * `d`: A dictionary that is expected to have a key called 'type' which specifies the type of the returned value.
```

Comment:
if thats not mintlify generated content ill eat my sock!

Reply:
So eat it :trollface:
For the outputs it was generated and I simplified/changed it a bit so the same can be used for all functions

Reply:
damn, at least theres tacos on my socks!